params_and_main.py

```diff
@@ -88,13 +88,22 @@
 large_file = False # If predicted probabilities should be stretched to int8 to increase storage capacity
 max_empty = 0.2  # Maximum no data area in created image crops
 ARCHITECTURE = xresnet34 #xresnet34
-transforms = [Dihedral(0.5),  # Horizontal and vertical flip
-              Rotate(max_deg=180, p=0.5),  # Rotation in any direction possible
-              Brightness(0.2, p=0.5),
-              Contrast(0.2, p=0.5),
-              Saturation(0.2),
-              Normalize.from_stats(*imagenet_stats)]
-transforms = None
+aug_pipe = A.Compose([
+            A.HorizontalFlip(p=0.5), # Applies a horizontal flip to the image with a probability of 0.5.
+            A.VerticalFlip(p=0.5), # Applies a vertical flip to the image with a probability of 0.5.
+            A.ShiftScaleRotate(p=0.5), # Randomly applies affine transforms: translation, scaling and rotation in one call with a probability of 0.5.
+            A.RandomBrightnessContrast( # Randomly changes brightness and contrast of the image with a probability of 0.5.
+                brightness_limit=(-0.1,0.1), 
+                contrast_limit=(-0.1, 0.1), 
+                p=0.5
+            ),
+            A.CoarseDropout(p=0.5), # Randomly masks out rectangular regions in the image with a probability of 0.5.
+
+]) # For more Augmentation options: https://github.com/albumentations-team/albumentations/tree/main#i-am-new-to-image-augmentation
+
+Num = 2 # Number of the augmented images minus from the totla number of batch size (default is 2).
```

Comment:
maybe a more descriptive variable names like 'n__transforms' or 'n_transform_imgs'

Reply:
yes, good idea, done!

Reply:
i would even go for percentage. So in relation to the batch_size. what do you think? otherwise it is a bit random when using different batch sizes e.g. 4, 8, 16, ...